crates/oxc_diagnostics/src/reporter/snapshots/oxc_diagnostics__reporter__json__tests__no_source.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/oxc_diagnostics/src/reporter/json.rs
+expression: pretty_output
+---
+[
+  {
+    "message": "Something happened!",
+    "severity": "warning",
+    "help": "Try something else.",
+    "labels": [
+      {
+        "span": {
+          "offset": 0,
+          "length": 1
+        }
+      }
+    ],
+    "related": []
+  }
+]
```

Comment:
Does this conform to what eslint produces? The format I had before is exact to what eslint produces.

https://github.com/eslint/eslint/blob/main/tests/lib/cli-engine/formatters/json.js